internal/cmd/environment/command_describe.go

```diff
@@ -1,11 +1,11 @@
 package environment
 
 import (
-	presource "github.com/confluentinc/cli/internal/pkg/name-conversions"
 	"github.com/spf13/cobra"
 
 	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
 	"github.com/confluentinc/cli/internal/pkg/errors"
+	nameconversions "github.com/confluentinc/cli/internal/pkg/name-conversions"
```

Comment:
I might have to keep this in a separate package to prevent a circular import with `dynamic_context` and `resource` later on, but if I can combine it with `resource` I will.